realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -43,8 +43,11 @@
     private static final long PRIMARY_KEY_FIELD_COLUMN_INDEX = 1;
     private static final long NO_PRIMARY_KEY = -2;
 
+    public static final boolean NULLABLE = true;
```

Comment:
Small thing. Personally I like `public static` variables at the top of the file, followed by `private static` instead of the other way around.